terraform/development/main.tf

```diff
@@ -149,6 +149,7 @@ module "simple_server_sandbox" {
   load_balancer_arn_suffix   = module.simple_networking.load_balancer_arn_suffix
   host_urls                  = ["api-sandbox.simple.org", "dashboard-sandbox.simple.org"]
   create_redis_instance      = true
+  create_redis_cache_instance = true
```

Comment:
Should we call one `create_cache_redis_instance` and the other `create_sidekiq_redis_instance`?